modules/libcom/src/osi/osiSock.h

```diff
@@ -163,6 +165,16 @@ typedef struct osiSockAddrNode {
     osiSockAddr addr;
 } osiSockAddrNode;
 
+typedef struct osiSockNet {
+    epicsUInt32 addr; /* network byte order */
+    epicsUInt32 mask; /* network byte order */
```

Comment:
In 2019 I don't think it is acceptable to assume that all network addresses are IPv4 addresses.  I'm not asking for the `AF_INET6` case to be implemented now, but I will insist that the `osiSockAddr` abstraction be used in all public APIs.